vivarium/processes/multibody_physics.py

```diff
@@ -83,35 +83,34 @@ def daughter_locations(parent_location, parent_values):
 
 
 class Multibody(Process):
-    """
-    A multi-body physics process using pymunk
-
-    To run with animation on set animate: True, and use the TKAgg matplotlib backend:
-    > MPLBACKEND=TKAgg python vivarium/processes/multibody_physics.py
+    """A multi-body physics process that tracks agent locations and simulates collisions
+        and forces.
 
-    Ports:
-
-        * ``agents``: The store containing all agent sub-compartments. Each agent in
-          this store has values for location, angle, length, width, mass, thrust, and torque.
+    :term:`Ports`:
+    * ``agents``: The store containing all agent sub-compartments. Each agent in
+      this store has values for location, angle, length, width, mass, thrust, and torque.
 
     Arguments:
         initial_parameters(dict): Accepts the following configuration keys:
 
-            * **jitter_force**: force applied to random positions along agent
-              bodies to mimic thermal fluctuations. Produces Brownian motion.
-            * **agent_shape** (:py:class:`str`): agents can take the shapes
-              ``rectangle``, `segment```, or ``circle``.
-            * **bounds** (:py:class:`list`): size of the environment in
-              micrometers, with [x, y].
-            * **mother_machine** (:py:class:`bool`): if True, mother machine
-              barriers are introduced.
-
-    Notes:
-        * rotational diffusion in liquid medium with viscosity = 1 mPa.s: Dr = 3.5+/-0.3 rad^2/s
-          (Saragosti, et al. 2012. Modeling E. coli tumbles by rotational diffusion.)
-        * translational diffusion in liquid medium with viscosity = 1 mPa.s: Dt=100 micrometers^2/s
-          (Saragosti, et al. 2012. Modeling E. coli tumbles by rotational diffusion.)
-
+        * **jitter_force**: force applied to random positions along agent
+          bodies to mimic thermal fluctuations. Produces Brownian motion.
+        * **agent_shape** (:py:class:`str`): agents can take the shapes
+        ``rectangle``, `segment```, or ``circle``.
+        * **bounds** (:py:class:`list`): size of the environment in
+          micrometers, with [x, y].
```

Comment:
I don't think anyone would get confused, but I like to add formatting to text that shouldn't be treated like prose, for example this list, just to make clear that it's not normal text

```suggestion
          micrometers, with ``[x, y]``.
```